public/js/Player.js

```diff
@@ -152,7 +163,7 @@ class Player extends GameObject {
 		this._y = Math.round(Math.random()*(Constants.gameHeight-40)) + 20;
 		this._bulletCount = 1;
 		this._alive = true;
-		this._score = 0;
+		//this._score = 0;
```

Comment:
we aren't removing it -- we're just not resetting the player's score in between deaths (score persists)